zoo-css-challenge/style.css

```
@@ -126,12 +135,28 @@ footer {
   color: #ef6f6c;
 }
 
+p:first-of-type{
```

```suggestion
.giraffe p:first-of-type{
```
We only want the first paragraph in the giraffe section to be bigger, so we can change this to `.giraffe p:first-of-type` so that the style is only applied to the giraffe section